src/pages/DebugChatPage.tsx

```
@@ -0,0 +1,230 @@
+
+  const resetState = () => {
+    clearTimers();
+    activeRunRef.current = null;
+    setMessages([]);
+  };
+
+  useEffect(() => {
+    const objectUrls = objectUrlsRef.current;
+    return () => {
+      clearTimers();
+      objectUrls.forEach((url) => URL.revokeObjectURL(url));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)